examples/minimal/main.tf

```
@@ -64,8 +63,10 @@ resource "google_compute_subnetwork" "test" {
   ip_cidr_range = "10.2.0.0/16"
 }
 
+# [START cloudloadbalancing_int_tcp_udp_gce]
 module "test_ilb" {
-  source       = "../../"
+  source       = "GoogleCloudPlatform/lb-internal/google"
+  version      = "2.3.0"
```

```suggestion
  version      = "~> 4.0"
```
Remember to always check the CHANGELOG for the latest up to date release: https://github.com/terraform-google-modules/terraform-google-lb-internal/blob/master/CHANGELOG.md